lib/src/cli/metrics/metrics_command.dart

```diff
@@ -227,7 +227,7 @@ Future<Digest> generateBuilderId() async {
       final regex = RegExp(macOSMachineIdRegEx, dotAll: true);
       return regex.firstMatch(machineId)?.group(1); // extract IOPlatformUUID
     } else if (Platform.isWindows) {
-      final regex = RegExp(r'\s*MachineGuid\s*\w*\s*([[:alnum:]-]+)', dotAll: true);
+      final regex = RegExp(r'\s*MachineGuid\s*\w*\s*([A-Za-z0-9-]+)', dotAll: true);
```

Comment:
Yes, dart regular expressions are based on [js](https://ecma-international.org/ecma-262/9.0/#sec-regexp-regular-expression-objects), not PCRE.